Tests/SentryTests/SentryCrash/SentryFrameInAppLogicTests.swift

```diff
@@ -0,0 +1,216 @@
+import XCTest
+
+class SentryFrameInAppLogicTests: XCTestCase {
+    
+    private class Fixture {
+        
+        func getSut(inAppIncludes: [String] = [], inAppExcludes: [String] = [] ) -> SentryFrameInAppLogic {
+            
+            return SentryFrameInAppLogic(
+                inAppIncludes: inAppIncludes,
+                inAppExcludes: inAppExcludes
+            )
+        }
+    }
+    
+    private let fixture = Fixture()
+    
+    func testNotInApp() {
+        XCTAssertFalse(fixture.getSut().is(inApp: "a/Bundle/Application/a"))
+        XCTAssertFalse(fixture.getSut().is(inApp: "a.app/"))
```

Comment:
M: create a local var for `fixture.getSut()`, so you don't need to instantiate the `SentryFrameInAppLogic` twice, this would be for all the test cases